src/pages/docs/oauth/flow/request-token.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: OAuth Request Token
+description: This describes the request token step
+tags:
+  - ShipEngine Connect
+  - OAuth
+---
+
+# Request Token
+
+After the end-user grants the Application access the integration redirects back
+to MonoAuth with a `code`. Monoauth makes a server-server request to the 3rd
+party to exchange the code for an access token.
+The `request_token` component of the DSL describes how MonoAuth should
+construct and execute that request.
+
+:brain: See the [OAuth 2.0 RFC - Access Token Request](https://tools.ietf.org/html/rfc6749#section-4.1.3)
+for more information on what standards are expected.
+
+```json
+"request_token": {
+    "url_template": "http://localhost:3002/access/token/request",    
+    "method": "POST",
+    "headers": [
+        {
+            "name": "Content-Type",
+            "value": "application/x-www-form-urlencoded"
+        }
+    ],
+    "body": [
+      {
+        "name": "grant_type",
+        "value": "authorization_code"
+      },
+      {
+        "name": "client_id",
+        "value": "{installation:client_id}"
+      },
+      {
+        "name": "client_secret",
+        "value": "{installation:client_secret}"
+      },
+      {
+        "name": "code",
+        "value": "{callback:code}"
```

Comment:
good opportunity to describe and link to the `{callback:}` expression documentation